### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by minibikemike** [allenh1/reef_moonshiners#16]
Only bug I see so far is the keyboard pops back up after hitting the done button.

**Comment by allenh1** [comment]
@minibikemike yep -- I'm not entirely sure what causes this, but it's pretty annoying to me as well. On my phone I have a small down arrow I can press to dismiss the keyboard, but the "done" button does just close and reopen the keyboard :roll_eyes:.

Thank you very much for reporting!
